data/incipias/incipias outfits.txt

```diff
@@ -0,0 +1,297 @@
+	"velocity scale" 0.
+
+outfit "Star Tail Cell"
+	category "Ammunition"
+	cost 500
+	thumbnail "outfit/star tail cell"
+	"mass" .1
+	"star tail capacity" -1
+	licenses
+		"Hicemus Conflict"
+	description "The Star Tail needs an extreme amount of energy that the Incipias are not able to create in a short amount of time. To accommodate this, the Star Tail Cell contains an energy-rich substance that releases its energy when fired. One of these cells holds enough energy to fire exactly one shot."
+
+outfit "Star Tail Storage"
+	category "Ammunition"
+	cost 500
+	thumbnail "outfit/star tail cell storage"
+	"mass" 10
+	"star tail capacity" 20
+	licenses
+		"Hicemus Conflict"
+	description "Because Star Tail Cells are highly volatile, they need a robust storage system. This radial design is able to withstand minor explosions by using a variant of a shield that can be deployed temporarily."
+	description "One layer of this easily expandable system is already shipped with the gun itself. To increase storage space, more layers are simply stacked on top."
+
+# Shields/Energy:
+outfit "Gas-Class Shield"
+	category "Systems"
+	thumbnail "outfit/gas class shield"
+	mass 60
+	"outfit space" -60
+	"shield generation" .1
+	"energy generation" 4.2
+	"heat generation" 5
+	description "Shield technology is one of the most recent developments of the Incipias. While this shield appears similar to a human shield device, it works on a completely different basis. In its core hydrogen is burned, providing energy for regeneration. This proved so effective that it became the main power source of all of their ships."
+
+outfit "Liquid-Class Shield"
+	category "Systems"
+	thumbnail "outfit/liquid class shield"
+	mass 80
+	"outfit space" -90
+	"shield generation" .25
+	"energy generation" 6.1
+	"heat generation" 6
+	description "This more advanced shield generator burns a liquid instead of a gas. This raises the efficiency considerably but also requires more space, making this outfit popular only on larger ships."
+
+outfit "Metallic Hydrogen Cell"
+	category "Power"
+	thumbnail "outfit/metallic hydrogen cell"
+	mass 20
+	"outfit space" -20
+	"energy capacity" 500
+	description "Hydrogen energy storage was discovered early by the Incipias because of the great amount of metallic hydrogen on their homeworld. Because Incipias ships usually access their energy at the time it is produced, only a small amount of storage is needed. Most of their ships have only one cell installed as a consequence."
+
+# Engines:
+outfit "MH Blue-Class Engine"
+	category "Engines"
+	thumbnail "outfit/mh blue-class engine"
+	"mass" 27
+	"outfit space" -27
+	"engine capacity" -27
+	"flare sprite" "effect/incipias flare/metallic hydrogen small"
+		"frame rate" 24
+	"flare sound" "plasma huge"
+	"thrust" 24
+	"thrusting energy" 2
+	"thrusting heat" 2.5
+	description "In Incipian culture, blue is the weakest color. An engine labeled with blue does not bode well for the speed of any ship using it."
```

Comment:
I think it should be this way?
```suggestion
	description "In the Incipian culture, blue is the weakest color. An engine labeled as blue does not bode well for the speed of any ship using it."
```
Nice ending, by the way 😉

Reply:
I think the the is unnecessary

Reply:
Then you can resolve this.

Reply:
```suggestion
	description "In Incipian culture, blue is the weakest color. An engine labeled as blue does not bode well for the speed of any ship using it."
```